Makefile

```diff
@@ -12,13 +12,6 @@ BINARY_NAME ?= newrelic-infra-operator
 
 LD_FLAGS ?= "-extldflags '-static'"
 
-ifeq (, $(shell which golangci-lint))
-GOLANGCI_LINT ?= go run -modfile=tools/go.mod github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
can get rid of https://github.com/newrelic/newrelic-infra-operator/blob/main/tools/tools.go and all horrible dependencies?

Reply:
Sure! I don't know why I missed it in this repo...

Reply:
Done: 801dedbd12ae06f475cdc4cb0ab5367160eaa6b3